### ISSUE TITLE
Base Path applied multiple times in v0.12.1132?

### ISSUE THREAD

**Issue by mad-tunes** [Jackett/Jackett#6572]
Provide a description of the feature request or bug, the more details the better.

I've had Jacket running well on Windows Server 2019 for months, as a windows service using /jackett as a Base Path and NGinx reverse proxy. Sonarr, radarr & lidarr all access Jackett using 127.0.0.1:9117/jackett internally, and I can get to it using domain/jackett from the outside world.

Jacket usually updates itself without issue, but today it killed itself updating from v0.12.1115 to v0.12.1132. (checked with 0.12.1133.0 too and still get the same).
Trying to browse to it any way I could think of redirected to http://ExtDomainOrIntIP/Jackett/jackett/UI/Login?ReturnUrl=%2FJackett%2FUI%2FDashboard

Forgetting my previous setup, I started with a blank folder and extracted 0.12.1132 in to it.
All is well until I set a Base Path Override of /jackett and restarted.
As soon as I did that, browsing to http://localhost:9117 or http://127.0.0.1:9117 works OK but browsing to http://192.168.1.1:9117 from another machine on the same network or to domain/jackett from outside leaves it pointing at DomainOrIPandPort/jackett/jackett/UI/Login?ReturnUrl=%2Fjackett%2FUI%2FDashboard ..and broken
uptimerobot.com reports 404 when watching domain/jackett

If I remove the /jacket from the config file and restart, everything works as it should but I then cant use it through NGinxs reverse proxy. (which is set as here: https://github.com/Jackett/Jackett#running-jackett-behind-a-reverse-proxy)

If I roll back to the previous version, things behave fine with or without the Base Path Override set.
Nothing unusual's logged during startup or the strange redirection.

Can anyone spot what I'm doing wrong, or have I found a problem?
Might this be related to [Issue 4765](https://github.com/Jackett/Jackett/issues/4765)/[pul 6561](https://github.com/Jackett/Jackett/pull/6561)?
I didn't used to find the behaviour mentioned in Issue 4765, It seemed to work perfectly with my nginx reverse proxy...

(At the bottom of your Jackett Dashboard page) **Jackett version**:  0.12.1132/0.12.1133.0

-------------------------------

**Comment by fryfrog** [comment]
I'm seeing the double base url too, `/jackett/jackett` on my Arch Linux install that was presumably working fine in the last version or two.

**Comment by nickmac88** [comment]
ditto! probably a bug

**Comment by AlessandroBagnoli** [comment]
Had the same bug. A service restart of nginx and jackett solved the issue for me.

**Comment by ngosang** [comment]
@garfield69 Please, revert #6561
I have tested it carefully and it works fine in my installation. I suppose some users have a different Nginx configuration and this change is causing troubles.

**Comment by mad-tunes** [comment]
I tried to exclude NGinx and still got /jackett/jackett with nginx stopped and while using local hostnames/IPs that don't go through it.

There may be a difference between accessing it using 127.0.0.1 and the machines LAN IP. Are both OK for you?

**Comment by garfield69** [comment]
reversion of 642bed1ef429a0d661cd1871799530a19bd7c037 included in jackett  0.12.1136

**Comment by mad-tunes** [comment]
Updated to 1136 and still running fine.
Thanks

**Comment by ngosang** [comment]
It's reverted but I will let all the details anyway.

Nginx config:
```
    location /jackett/ {
        # https://github.com/Jackett/Jackett#running-jackett-behind-a-reverse-proxy
        proxy_buffering off;
        proxy_read_timeout 300;
        proxy_pass http://127.0.0.1:9117/;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_set_header X-Forwarded-Host $http_host;
        proxy_redirect off;
    }
```
In Jackett config => Base Path Override = /jackett

I can access from the domain.com/jackett and from 127.0.0.1.
* https://mydomain.org/jackett/UI/Dashboard
* http://127.0.0.1/jackett/UI/Dashboard

**Comment by fryfrog** [comment]
Your `proxy_pass` is "wrong", it *should* be `http://127.0.0.1:9117/jackett` since you set the base path override. But I think jackett happily accepts requests from `/` too, so it works. And that is probably why it works for you and no one else.

**Comment by mad-tunes** [comment]
I'd always thought similar.. but the example doesnt have that so I didn't either:
location /jackett {
	proxy_pass http://127.0.0.1:9117;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $scheme;
	proxy_set_header X-Forwarded-Host $http_host;
	proxy_redirect off;
}

..I have now

**Comment by fryfrog** [comment]
The example probably presumes you're not setting the base url? You don't *need* to, it just makes it nicer looking.

**Comment by ngosang** [comment]
@fryfrog I copied the configuration from here. https://github.com/Jackett/Jackett#running-jackett-behind-a-reverse-proxy For some reason it didn't work for me and I added the base path override. That change fixed something but caused #4765
Just now, I removed the base path override and everything is working. Thank you.
@all sorry for my mistake.

**Comment by fryfrog** [comment]
You didn't make a mistake, something about reverse proxy *did* get broken. :)
